File: ComInLanDroid/cominlan/src/main/java/com/onballgroup/cominlan/model/protocol/IClientProtocol.java
package com.onballgroup.cominlan.model.protocol;

import com.onballgroup.cominlan.model.Base.IBaseModel;

/**
 * Created by Phong Le on 4/20/2016.
 */
public interface IClientProtocol extends IBaseModel {
    ClientCommand getCommand();
    Object getData();
}

File: ComInLanWindows/ComInLan/Model/Protocol/IClientProtocol.cs
using ComInLan.Model.Base;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ComInLan.Model.Protocol
{
	public interface IClientProtocol : IJson
	{
		ClientMessage Message { get; }
	}
}
